components/catalogs/back/src/Application/Mapping/ValueExtractor/Extractor/ValueExtractorInterface.php

```diff
@@ -14,6 +14,18 @@
  */
 interface ValueExtractorInterface
 {
+    public const SUPPORTED_SOURCE_TYPE_DATE = 'pim_catalog_date';
```

Comment:
```suggestion
    public const SOURCE_TYPE_ATTRIBUTE_DATE = 'pim_catalog_date';
```
It's only a suggestion, it's not required.

1) `SUPPORTED_` can be removed. The constant is more "we have this source type or this source type", and the method `getSupportedSourceType` is asking, which one do you support ?"
2) For attribute types, I'd rather that we specify that they are in fact attributes.

Reply:
Ahah I really hesitated to add `ATTRIBUTE` in the constant name

Reply:
We did it for criteria https://github.com/akeneo/pim-community-dev/tree/master/components/catalogs/front/src/components/ProductSelection/criteria